bentoml/_internal/runner/container.py

```diff
@@ -82,13 +86,15 @@ def batch_to_payloads(cls, batch: BatchType, batch_axis=None) -> t.List[Payload]
 
 class NdarrayContainer(DataContainer["tnp.ndarray", "tnp.ndarray"]):
     @classmethod
-    def singles_to_batch(cls, singles, batch_axis=0):
+    def singles_to_batch(
+        cls, singles: t.Sequence["tnp.ndarray"], batch_axis: t.Optional[int] = 0
```

Comment:
this kinda also depends on the stubs PR to get merged as well, but in numpy 1.20+:
```
... singles: t.Sequence["tnp.ndarray[t.Any, np.dtype[t.Any]"], ...
```

Reply:
```suggestion
        cls, singles: t.Sequence["tnp.ndarray[t.Any, np.dtype[t.Any]"], batch_axis: t.Optional[int] = 0
```